src/main.rs

```diff
@@ -0,0 +1,86 @@
+use actions_github::context::get_context;
+use actions_github::core::{get_input, set_output};
+use actions_github::logger;
+use anyhow::Result;
+use octocrab::Octocrab;
+
+#[tokio::main]
+async fn main() {
+    let ctx = match get_context() {
+        Ok(context) => context,
+        Err(error) => panic!("{}", error),
+    };
+    let org: String = ctx.repo.owner;
+
+    let team_name: String = get_variable("team", true);
+
+    let token: String = get_variable("ACCESS_TOKEN", true);
+
+    let crab = Octocrab::builder().personal_token(token).build();
+    octocrab::initialise(crab.unwrap());
+
+    let team = fetch_team(org, &team_name).await.unwrap();
+
+    if team.is_empty() {
+        panic!(
+            "No users where found while searching for the team {}",
+            &team_name
+        );
+    }
+
+    logger::debug_log(format!("Obtained data from {} users", team.len()).as_str());
+    if let Err(err) = set_output(
+        "usernames",
+        team.clone()
```

Comment:
The `.clone()` here seems redundant

Reply:
This is the only one I can't do because clippy throws:

>  Checking list-team-members v0.1.0 (/Repositories/list-team-members)
> error[E0507]: cannot move out of `member.username` which is behind a shared reference
>   --> src/main.rs:33:27
>    |
> 33 |             .map(|member| member.username)
>    |                           ^^^^^^^^^^^^^^^ move occurs because `member.username` has type `std::string::String`, which does not implement the `Copy` trait

Any ideas?

Reply:
You sure that you've removed `.clone()` from `team.clone()`, not from `member.username.clone()`?

Reply:
I was not sure cleary 😅.

Now it works 😄